<launch>
  <arg name="param_folder"   default="map_nav_params"  doc="map_nav_params or odom_nav_params"/>

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">

    <rosparam file="$(find real_nav)/params/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find real_nav)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />

    <rosparam file="$(find real_nav)/params/$(arg param_folder)/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find real_nav)/params/$(arg param_folder)/global_costmap_params.yaml" command="load" />

<!--    <rosparam file="$(find real_nav)/params/base_local_planner_params.yaml" command="load" />-->
    <rosparam file="$(find real_nav)/params/base_local_planner_params_clearpath.yaml" command="load" />
    <rosparam file="$(find real_nav)/params/move_base_params.yaml" command="load" />
    <rosparam file="$(find real_nav)/params/global_planner.yaml" command="load" />

    <param name="base_global_planner" type="string" value="global_planner/GlobalPlanner" />
<!--    <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS"/>-->
    <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS"/>

    <remap from="odom" to="odometry/filtered" />
  </node>

</launch>